internal/transport/http_test.go

```diff
@@ -69,7 +69,9 @@ func getFreePort() (int, error) {
 	if err != nil {
 		return 0, err
 	}
-	defer l.Close()
+	defer func() {
+		_ = l.Close()
```

Comment:
No `return 0, err` here?

Reply:
It is only unit test. We do not write to `l`. There is zero chance of any serious error. It can be ignored.